src/core/propertyTypes.js

```diff
@@ -85,7 +85,7 @@ function selectorParse (value) {
 function selectorAllParse (value) {
   if (!value) { return null; }
   if (typeof value !== 'string') { return value; }
```

Comment:
I think the first case may be for `{default: ''}` so we don't need to run that.

The second case is if an array or NodeList is passed in yeah.